src/_pytest/raises.py

```diff
@@ -43,6 +43,9 @@
         default=BaseException,
         covariant=True,
     )
+
+    # use short name because it shows up in docs
```

Comment:
Minor, but we usually tend to use proper grammar in comments:

```suggestion
    # Use short name because it shows up in docs.
```

This makes it more pleasant to read/write, because often, without this practice, one will end up with comments with one line that do not end up with `.`, while others with more than one line that do end in `.`... always using proper grammar prevents those little inconsistencies and is one less thing to keep in mind. Plus it helps when you need to write sentences where this does matter (like docs).